integration/setup.go

```diff
@@ -0,0 +1,161 @@
+package integration
+
+import (
+	"io/ioutil"
+	"path/filepath"
+	"runtime"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/xichen2020/eventdb/services/eventdb/serve"
+	"github.com/xichen2020/eventdb/storage"
+
+	xconfig "github.com/m3db/m3x/config"
+	"github.com/m3db/m3x/instrument"
+	"github.com/m3db/m3x/log"
+	"github.com/uber-go/tally"
+)
+
+const (
+	gracefulShutdownTimeout = 15 * time.Second
+)
+
+var (
+	logger = log.NullLogger
+
+	// basePath contains the path to this directory.
```

Comment:
Hm I don't like the hack.

Can you just assign the yaml file to a static string var and unmarshal from there? Something like
```
var testConfig1 =
...
```
and unmarshal directly from there?

I don't think it's important to test the file loading logic in the integration test.